examples/workers/l4/vector_add_mixed_l3/main.py

```
@@ -0,0 +1,349 @@
+        remote_buffers.extend(remote_handles)
+        for handle, array in zip(
+            remote_handles,
+            (
+                _make_array(6.0),
+                _make_array(7.0),
+                _make_array(0.0),
+                _make_array(8.0),
+                _make_array(9.0),
+                _make_array(0.0),
+            ),
+        ):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)